test/expected/plan_ordered_append.out

```diff
@@ -0,0 +1,237 @@
+-- This file and its contents are licensed under the Apache License 2.0.
```

Comment:
I would like to see a diff test for this akin to plan_expand_hypertable_results_diff

Reply:
done